### PR TITLE
Added do you need Service mesh section

### LINKED ISSUE #1207: [NG] Add new sections
**Desired Situation**

Add the following, new component sections on Layer-ng:

- [x]  Do I need a service mesh?
- [x]  What is a service mesh?

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by OmkarPh**
Signed-off-by: OmkarPh <omkarphansopkar@gmail.com>

**Description**

This PR fixes #1207 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by OmkarPh** [comment]
@Jashpatel1 ^^

**Review by Jashpatel1 — COMMENTED** [review]
Add some padding for mobile view, and reduce the size of Let's find out circle in mobile view. That would be all I think.

**Comment by OmkarPh** [comment]
What do you think about inner shadow on clicking btn?

File: src/sections/DoYouNeedService/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import { Link } from "gatsby";
+
+import DoYouNeedWrapper from "./doYouNeedService.style";
+
+const DoYouNeedService = ({onClick}) => {
+    return (
+        <DoYouNeedWrapper>
+            <div id="content">
+                <div id="text">
+                    <span className="heading">Do You Need A <br/>Service Mesh?</span>
```
Comment (Jashpatel1) [inline src/sections/DoYouNeedService/index.js:11]: Don't use <br> here. It affects the mobile view. Instead, define the width of this element to fit only the number of words you wish to see in one line.

Comment (OmkarPh) [reply src/sections/DoYouNeedService/index.js:11]: I agree on that too ✌️

File: src/sections/DoYouNeedService/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+                    <br/><br/>
+                    <span className="description">Let us work you through</span>
```
Comment (leecalcote) [inline src/sections/DoYouNeedService/index.js:13]:
```suggestion
                    <span className="description">Let's find out.</span>
```

Comment (OmkarPh) [reply src/sections/DoYouNeedService/index.js:13]: If it Ok to have same description and button text, I'd do that ✌️

Comment (OmkarPh) [reply src/sections/DoYouNeedService/index.js:13]: Fine ✌️

**Review by Jashpatel1 — APPROVED** [review]
@OmkarPh great work here. Just remove it from the home page.

**Review by Jashpatel1 — APPROVED** [review]

**Pull request merged**